examples/user_guide/12-Custom_Interactivity.ipynb

```diff
@@ -239,8 +229,14 @@
     "    summed = area.dimension_values('y').sum() * 0.015  # Numeric approximation\n",
     "    return (area * curve * hv.VLine(limit) * hv.Text(limit + 0.8, 2.0, '%.2f' % summed))\n",
     "\n",
-    "hv.DynamicMap(integral, streams=[streams.Stream.define('Time', time=1.0)(), \n",
-    "                                 streams.PointerX().rename(x='limit')])"
+    "integral_streams = [\n",
+    "    streams.Stream.define('Time', time=1.0)(),\n",
+    "    streams.PointerX().rename(x='limit')]\n",
+    "\n",
+    "integral_dmap = hv.DynamicMap(integral, streams=integral_streams)\n",
```

Comment:
Might be nice to call the dynamicmap `integral` and the callback something else...e.g `integral_X` or `X_integral` where X is descriptive..